src/libraries/Common/src/System/IO/FileSystem.Attributes.Windows.cs

```diff
@@ -40,6 +51,15 @@ public static bool FileExists(string fullPath)
                 ((data.dwFileAttributes & Interop.Kernel32.FileAttributes.FILE_ATTRIBUTE_DIRECTORY) == 0);
         }
 
+        public static (int fileAttributes, int lastError) GetFileAttributes(string fullPath)
```

Comment:
this and DirectoryExistsSlim return last error in inconsistent ways.

can you just have a single method, that returns `(bool fileExists, bool directoryExists)` ? (or `(bool fileExists, bool directoryExists, int lastError)` if necessary). that would also make the callers simpler.

Reply:
I had to remove this in commit 8cfca8641f11d052c8c81a1cb3a415104c59dd13 as it caused the test to fail.
But this part is only executed if the directory already exists.